docs/developers/celestia-node.mdx

```diff
@@ -289,22 +289,22 @@ Error: nodebuilder/share: interval must be positive
 You can re-initialize your node with the following command:
 
 ```sh
-rm -rf ~./celestia-light-arabica-1 && celestia light init
+rm -rf ~./celestia-light-arabica-2 && celestia light init --p2p.network arabica
 ```
 
 Your output will look similar to below:
 
 ```sh
-@ ~ % rm -rf ./celestia-light-arabica-1 && celestia light init
-2022-11-01T22:23:19.581+0100	INFO	node	nodebuilder/init.go:20	Initializing Light Node Store over '/Users/joshstein/.celestia-light-arabica-1'
-2022-11-01T22:23:19.582+0100	INFO	node	nodebuilder/init.go:51	Saving config	{"path": "/Users/joshstein/.celestia-light-arabica-1/config.toml"}
+@ ~ % rm -rf ./celestia-light-arabica-2 && celestia light init
```

Comment:
delete this line as you're saying "your output" so you're just showing the logs after running this command and you already ran it on line 292

Reply:
✅ done - thank you